ts-typed/player-config.d.ts

```diff
@@ -187,4 +186,22 @@ declare namespace KalturaPlayerTypes {
     debug?: boolean;
     log?: Log;
   }
+
```

Comment:
Lets create a generic type which is OVPMediaInfo|OTTMediaInfo to keep the provider type transparent

Reply:
From what I saw we can't do it in the same way the flow definition does.
What i can do it make interface MediaInfo and have 2 separate interfaces extending it.

Reply:
ok